app/uk/gov/hmrc/traderservices/views/ImportQuestionsView.scala.html

```diff
@@ -0,0 +1,83 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.traderservices.models.ImportQuestions
+@import uk.gov.hmrc.traderservices.views.ImportQuestionsViewContext
+
+@this(
+        govukLayout: uk.gov.hmrc.traderservices.views.html.templates.GovukLayoutWrapper,
+        forms: uk.gov.hmrc.traderservices.views.components.forms,
+        html: uk.gov.hmrc.traderservices.views.components.html,
+        context: ImportQuestionsViewContext
+)
+
+@(form: Form[ImportQuestions], postAction: Call)(implicit request: Request[_], messages: Messages)
+
+@govukLayout(pageTitle = Some(messages("view.import-questions.title"))) {
+
+  @forms.errorSummary(form.errors)
+
+  @html.h1(messages("view.import-questions.heading"))
+
+  @html.p {
+    @messages("view.import-questions.description")
+  }
+
+  @forms.formWithCSRF(action = postAction, args = 'novalidate -> "novalidate") {
+
+    @forms.inputCheckboxes(
```

Comment:
I think we are going to use only radio buttons here, do we?

Reply:
I think that's the new plan but the current design attached to the ticket still shows checkboxes

Reply:
I see you've already switched to radios on the exports page. I'll do the same for imports then

Reply:
Done